src/Razor/src/Microsoft.NET.Sdk.Razor/GenerateAspNetCoreStaticAssetsManifest.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.Tasks
+{
+    public class GenerateAspNetCoreStaticAssetsManifest : Task
+    {
+        private const string ContentRoot = nameof(ContentRoot);
```

Comment:
consider not using `nameof` here. These are contracts and can't easily change. `nameof` is for things you might refactor later (and for feeling smart).

Reply:
I've done this all my life. The alternative is typos, but I'll change it.